### DISCUSSION TITLE
its not really an issue but issue/request, stereo audio

### DISCUSSION THREAD

**Discussion by geekositalia** [edumeet/edumeet#768] [Q&A]
Hi, im jo boschetti from geekos daw, opensuse project for professional audio. I believe that this software is the best for streaming and teaching, many people have asked me for stereo videoconferencing to work in the audio branch as it is done with zoom (not in real time), I would be proud to present this project, for this type of use , but in addition to deactivating the filters, the possibility of making the room stereo is required, and allowing users to choose stereo or mono audio, have you thought of a similar feature?

**Comment by so010** [comment]
Move to discussions ...

**Comment by so010** [comment]
You should be able to configure for stereo. Have a look into your application config file in /app/pub/config.js
	centralAudioOptions :
```
	{
		// will not eat that much bandwith thanks to opus
		sampleRate          : 48000, // default : 48000 and don't go higher
		// usually mics are mono so this saves bandwidth
		channelCount        : 1, // default : 1
		volume              : 1.0, // default : 1.0
		sampleSize          : 16, // default : 16
		// usually mics are mono so this saves bandwidth
		opusStereo          : false, // default : false
		opusDtx             : true, // default : true / will save bandwidth 
		opusFec             : true, // default : true / forward error correction
		opusPtime           : '20', // default : 20 / minimum packet time (3, 5, 10, 20, 40, 60, 120)
		opusMaxPlaybackRate : 48000 // default : 48000 and don't go higher
	},
```
You have to set channelCount to 2 (number of channels from sound source)
and opusStereo to true (will enable stereo in the audio codec)

**Comment by so010** [comment]
We want to make this configurable from client-side as well. Track this issue:
https://github.com/edumeet/edumeet/issues/549
